### PR TITLE
Added Basic Test for OrderController

### LINKED ISSUE #30: Crear test básico para la clase OrderController
Crear para la siguiente clase, su test correspondiente:

- `OrderController`

Usar como ejemplo el test de la clase `ExcludeDayController`.

- `ExcludeDayControllerTest`
- `ExcludeDayControllerTestUtil`

De ser necesario crear los métodos **fake** que hagan falta en la clase `TestUtil`.

Si hay dudas, preguntar.

### PR DESCRIPTION
**Pull request by rupeshpurum**

### CODE REVIEW

File: src/test/java/red/softn/npedidos/controller/OrderControllerTestUtil.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    public OrderControllerTestUtil(Gson gson) {
+        super(gson);
+    }
+    
+    @Override
+    protected Init<OrderRequest, OrderResponse, Integer> init() {
+        var request = new OrderRequest();
+        var response = new OrderResponse();
+        request.setId(TestUtil.fakeRandomInteger());
+        response.setId(request.getId());
```
Comment (nmarulo) [inline src/test/java/red/softn/npedidos/controller/OrderControllerTestUtil.java:22]: property `dateOrder` is missing

**Review by nmarulo — CHANGES_REQUESTED** [review]
Add the missing property for both instances.

File: src/test/java/red/softn/npedidos/controller/OrderControllerTestUtil.java (hunk 1 above)
Comment (rupeshpurum) [reply src/test/java/red/softn/npedidos/controller/OrderControllerTestUtil.java:22]: @nmarulo ,
Thanks for the feedback. dateOrder property is added for both Request and Response objects.

**Comment by nmarulo** [comment]
close this PR and review the new PR #49

**Pull request closed without merging**